examples/green-house-migration/routes/api/teamtailor_dashboard.py

```
@@ -0,0 +1,689 @@
+
+        return {
+            "success": True,
+            "data": processed_candidates,
+            "pagination": {
+                "page": page,
+                "page_size": len(processed_candidates),
+                "total": len(processed_candidates),
+            },
+        }
+
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)